Penteract/Assets/Scripts/DialogueManager.cpp

```diff
@@ -159,8 +207,21 @@ void DialogueManager::SetActiveDialogue(Dialogue* dialogue, bool runAnimation) {
 		uiComponents.clear();
 		uiColors.clear();
 		RetrieveUIComponents(activeDialogueObject);
+		GameController::isGameplayBlocked = dialogue->isBlocking;
+
+		// Camera Zoom In
+		if (cameraControllerScript) {
+			cameraControllerScript->ChangeCameraOffset(-2, 10, 6);
```

Comment:
Make this a variable so it's no longer a magic number and expose the offset so it can be tweaked from Editor